test/WatchdogTest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace AgDevelop\ForkingSupervisor\Test;
+
+use AgDevelop\ForkingSupervisor\Watchdog\Watchdog;
+use PHPUnit\Framework\TestCase;
+
+class WatchdogTest extends TestCase
+{
+
+    public function dataProvider()
+    {
+        return [
+            [new \DateTime("-11 seconds"), new \DateTime("-4 seconds"), 10, 5, true, true],
+            [new \DateTime("-9 seconds"), new \DateTime("-4 seconds"), 10, 5, false, false],
+            [new \DateTime("-11 seconds"), new \DateTime("-6 seconds"), 10, 5, true, true],
+            [new \DateTime("-9 seconds"), new \DateTime("-6 seconds"), 10, 5, false, true],
+        ];
+    }
+
+    /** @dataProvider dataProvider */
+    public function testShould(\DateTimeInterface $createdAt, \DateTimeInterface $lastOccupied,
+        int $maxAlive, int $maxUnocuppied, bool $shouldBeTerminated, bool $shouldExit)
```

Comment:
Tak szczerze to według mnie dziwnie jest to trochę przeniesione. Część parametrów została przeniesiona, a część nie. Według mnie albo dla każdego jest oddzielna linijka, albo wcale :D

Reply:
Krzychu, poprawiłem to ale myślę, że nie powinniśmy się angażować się w to aż tak bardzo. Zwracajmy tylko uwagę na łamanie reguł ustawionych php-cs-fixer'em.